Resources/Textures/Structures/Doors/turnstile.rsi/meta.json

```diff
@@ -0,0 +1,19 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "DrMelon",
```

Comment:
should be more descriptive

if you drew these for ss14 say Created by DrMelon (github) for SS14.